composer.json

```diff
@@ -10,24 +10,23 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0 || ^8.0",
-        "php-task/php-task": "^1.4",
-        "symfony/http-kernel": "^2.8 || ^3.4 || ^4.0 || ^5.0",
-        "symfony/dependency-injection": "^2.8 || ^3.4 || ^4.0 || ^5.0",
-        "symfony/expression-language": "^2.8 || ^3.4 || ^4.0 || ^5.0",
-        "symfony/config": "^2.8 || ^3.4 || ^4.0 || ^5.0",
-        "symfony/console": "^2.8 || ^3.4 || ^4.0 || ^5.0",
-        "symfony/process": "^2.8 || ^3.4 || ^4.0 || ^5.0",
-        "doctrine/orm": "^2.5"
+        "php": "^8.0 || ^8.1",
+        "php-task/php-task": "dev-master as 2.0",
```

Comment:
tag before merging

Reply:
@Prokyonn please update the composer.json https://github.com/php-task/php-task/releases/tag/2.0.0

Reply:
@wachterjohannes done